charts/ratify/templates/verifier.yaml

```diff
@@ -11,13 +11,15 @@ spec:
   name: notation
   artifactTypes: application/vnd.cncf.notary.signature
   parameters:
-    
     verificationCertStores:
       certs:
         {{- if .Values.akvCertConfig.enabled }}
         - certstore-akv
         {{- else }}
           {{- if .Values.notationCert }}
+            {{- if .Values.notationCerts }}
+            {{- fail "please specify notation certs with .Values.notationCerts, single certificate .Values.notationCert is deprecated, will soon be removed." }}
```

Comment:
Does the `fail` template function blocks the helm install? or is it just an error message?

Reply:
this will be blocking msg.  See output below.

![image](https://github.com/deislabs/ratify/assets/8160956/33998da5-b91d-46cd-b8d3-4e22491a62aa)